.github/workflows/test-environment.yml

```diff
@@ -152,6 +152,18 @@ jobs:
           echo "::add-mask::$ec_api_key"
           echo "TF_VAR_ec_api_key=$ec_api_key" >> $GITHUB_ENV
 
+      - name: Process BC version
+        id: remove-commit-hash
+        run: |
+          # Extract the stack version
+          stack_version="${{ inputs.elk-stack-version }}"
+
+          # Check if the version contains a commit hash, remove it
+          if [[ $stack_version =~ -[a-f0-9]+ ]]; then
+            cleaned_version=$(echo $stack_version | awk -F"-" '{print $1}')
+            echo "STACK_VERSION=$cleaned_version" >> $GITHUB_ENV
```

Comment:
Remove it's declaration on line 115

Reply:
Updates the declaration to support the working of the entire flow retroactively. The version with the hash is only required for Terraform to pin the version of the stack to be deployed.